e2e_tests/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+# This crate should NOT be upstreamed to crates.io; it is only intended for 
+# PARSEC testing purposes (both locally and on CI)
+[package]
```

Comment:
You can add [this](https://doc.rust-lang.org/cargo/reference/manifest.html#the-publish-field):
```toml
[package]
# ...
publish = false
```
to enforce it won't be done by mistake!

Reply:
Done!